src/tools/material.js

```diff
@@ -18,6 +20,19 @@ function scaleOneLatticeVector(material, key = 'a', factor = 1.0) {
 
 }
 
+/**
+ * Updates the size of a materials lattice using the minimumLatticeSize function.
+ * The new size of the material is calculated based on the materials basis.
+ * @param material {Material}
+ */
+function scaleLatticeToMakeNonPeriodic(material) {
+    material.lattice = new Lattice({
+        a: material.Basis.minimumLatticeSize,
```

Comment:
I realized i did not update this function to the proper one. It should call `minimumLatticeSize` function, which was implemented for updating the lattice in the `hooks.js` function for the web-app. It should also be used here. https://github.com/Exabyte-io/made.js/blob/feature/SOF-5215-ALD/src/basis/basis.js#:~:text=*/-,get%20minimumLatticeSize()%20%7B,%7D,-/**